From 5d2073205b8cd39b3ecbef17b127f2cd9c7f705f Mon Sep 17 00:00:00 2001
From: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Date: Fri, 16 Feb 2024 09:41:38 +0100
Subject: [PATCH 0682/1695] dts: stm32: enable HASH on stm32mp257f-dk

Set HASH status to "okay" in stm32mp257f-dk.dts

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 8bf54abbbd96bbc9d9b1c1f27ec0521cf186181d dts: stm32: enable HASH on stm32mp257f-dk]
Signed-off-by: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Change-Id: I81d8d3662aee4e172a8f42a59b502bbac83547bc
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/360596
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Domain-Review: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 core/arch/arm/dts/stm32mp257f-dk.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp257f-dk.dts b/core/arch/arm/dts/stm32mp257f-dk.dts
index e03e01594..35b529ddc 100644
--- a/core/arch/arm/dts/stm32mp257f-dk.dts
+++ b/core/arch/arm/dts/stm32mp257f-dk.dts
@@ -48,6 +48,10 @@
 	};
 };
 
+&hash {
+	status = "okay";
+};
+
 &hsi_calibration {
 	status = "okay";
 };
-- 
2.47.2

